modules/Story/Share/utils/htmlToText.ts

```diff
@@ -0,0 +1,15 @@
+'use server';
```

Comment:
In dev environment (unminified scripts), having it as a server action saves ~260kb. Not sure how much would it be in prod though.